tests/php/Core/Manifest/ManifestFileFinderTest.php

```diff
@@ -91,7 +91,7 @@ public function testIgnorePHPUnit5Tests()
         $finder = new ManifestFileFinder();
         $finder->setOption('name_regex', '/\.txt$/');
         $finder->setOption('ignore_tests', false);
-        $finder->setOption('ignore_ci_libraries', [Module::CI_PHPUNIT_FIVE]);
+        $finder->setOption('ignore_ci_configs', [Module::CI_PHPUNIT_FIVE]);
```

Comment:
```suggestion
        $finder->setOption('ignored_ci_configs', [Module::CI_PHPUNIT_FIVE]);
```
?

The `ignore_tests` above, from my perspective, reads as "do ignore tests yes/no", so "ignore ci configs" would also be yes/no, whereas "ignored ci config" would take a list of configs to ignore.

Reply:
Done.